sentry-android-timber/src/main/java/io/sentry/android/timber/SentryTimberTree.kt

```diff
@@ -227,6 +235,23 @@ public class SentryTimberTree(
     }
   }
 
+  /** Send a Sentry Logs */
+  private fun addLog(
+    sentryLogLevel: SentryLogLevel,
+    msg: String?,
+    throwable: Throwable?,
+    vararg args: Any?,
+  ) {
+    // checks the log level
+    if (isLoggable(sentryLogLevel, minLogsLevel)) {
+      val throwableMsg = throwable?.message
+      when {
```

Comment:
`m` Could there be both a `msg` and a `throwableMsg`? Would we want to add whichever we deem less important as an attribute on the log event?

Reply:
added the case doing `$msg\n$throwableMsg` as it's the same of what Timber does